src/registry_schemas/schemas/correction.json

```diff
@@ -170,6 +170,64 @@
               }
             }
           }
+        },
+        {
+          "if": {
+            "required": ["legalType"],
+            "properties": {
+              "legalType": {
+                "enum": [
+                  "CP"
+                ]
+              }
+            }
+          },
+          "then": {
+            "required": [
+              "contactPoint",
+              "business"
+            ],
+            "properties": {
+              "type": {
+                "type": "string"
+              },
+              "resolutionConfirmed": {
+                "type": "boolean"
+              },
+              "business": {
+                "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/business#/properties/business"
+              },
+              "contactPoint": {
+                "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/contactPoint"
+              },
+              "memorandumFileKey": {
+                "type": "string",
+                "title": "The Identifier for memorandum file in file server"
+              },
+              "memorandumFileName": {
+                "type": "string"
+              },
```

Comment:
is there a need to have `memorandumInResolution` as a property?

Reply:
thanks Argus, I removed it.

Reply:
I meant, don't we need to add `memorandumInResolution` as a property?

For example, if the checkbox below is set via the SR correction flow, we need to use/set the `memorandumInResolution` property right?

![image](https://github.com/bcgov/business-schemas/assets/6685223/a0220c94-72c8-4d76-8c10-625288128ab4)

Reply:
We need the property argus, you can include both the rules and memorandum in the SR at the same time.